<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BPTruBPM200Class</class>
 <widget class="QMainWindow" name="BPTruBPM200Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BPTruBPM200</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>691</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="OnyxStyleVerticleLayout">
     <item>
      <layout class="QHBoxLayout" name="UserIDHorizontalLayout">
       <item>
        <widget class="QLabel" name="UserIDLabel">
         <property name="text">
          <string>UserId:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="UserIDVal">
         <property name="text">
          <string>________</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ConfirmUserID">
         <property name="text">
          <string>Enter UserID to confirm:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ConfrimUserIDInput">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="InfoHorizontalLayout">
       <item>
        <layout class="QGridLayout" name="InstrumentGridLayout">
         <item row="2" column="0">
          <widget class="QLabel" name="FirmwareLabel">
           <property name="text">
            <string>Firmware</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="InstrumentTitle">
           <property name="text">
            <string>&lt;b&gt;Instrument&lt;/b&gt;</string>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="FirmwareVal">
           <property name="text">
            <string>---</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="StateVal">
           <property name="text">
            <string>---</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="StateLabel">
           <property name="text">
            <string>State</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="SpacerLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="CurrentMeasurementGridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="CurrentMeasurementTitle">
           <property name="text">
            <string>&lt;b&gt;Current Measurement&lt;/b&gt;</string>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="ReadingLabel">
           <property name="text">
            <string>Reading:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="CycleLabel">
           <property name="text">
            <string>Cycle (min):</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="ReadingVal">
           <property name="text">
            <string>---</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="CycleVal">
           <property name="text">
            <string>---</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="PressureLabel">
           <property name="text">
            <string>Cuff Pressure</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="PressureVal">
           <property name="text">
            <string>---</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="Buttons">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="StartButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="AddButton">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="StopButton">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ExitButton">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>OnStartClicked()</slot>
 </slots>
</ui>
